apps/dialtone-documentation/docs/.vuepress/theme/client.js

```diff
@@ -83,28 +87,42 @@ async function registerDialtoneVue (app) {
       dialtoneConstants[key] = module[key];
     } else if (key.endsWith('Directive')) {
       app.use(module[key]);
-    } else {
+    } else if (key.startsWith('Dt')) {
```

Comment:
One detail about this: we are also using the date functions `formatLong`, `formatMedium`, `formatShort`, `formatNoYear`, `formatNumerical` in `datepicker.md`. These are no longer part of dialtoneComponents, so they are not showing. Maybe they can be injected as a separate dependency `dateFunctions`.

Reply:
Sure, because those are not components, I'll add them.